infrastructure/transform/locals.tf

```diff
@@ -65,4 +65,12 @@ locals {
     ]
   ]))
 
+  data_source_connection_secrets = flatten([for connection in var.data_source_connections : {
+    name     = connection.name
+    fqdn     = "{{secrets/${databricks_secret_scope.secrets.name}/flowehr-dbks-${connection.name}-fqdn}}"
```

Comment:
could we have a comment about why they're in this absolutely nuts format?

Reply:
good point - I just moved the whole thing to where Spark config is, so hopefully it's more clear now